projects/packages/admin-ui/src/class-admin-menu.php

```diff
@@ -173,6 +173,26 @@ public static function add_menu( $page_title, $menu_title, $capability, $menu_sl
 		return 'jetpack_page_' . $menu_slug;
 	}
 
+	/**
+	 * Removes an already added submenu
+	 *
+	 * @param string $menu_slug   The slug of the submenu to remove.
+	 *
+	 * @return array|false The removed submenu on success, false if not found.
```

Comment:
This behavior is the same as that of [`remove_submenu_page`](https://developer.wordpress.org/reference/functions/remove_submenu_page/)